### ISSUE TITLE
feature: description for bookmarks

### ISSUE THREAD

**Issue by corani** [lalitmee/browse.nvim#4]
Rather than a simple list of URLs, it would be nice to be able to add a description for each URL.

**Comment by lalitmee** [comment]
@corani, not sure how useful it will be. If you have an idea of how to implement this, go ahead and raise the PR.

**Comment by corani** [comment]
It would be useful with long opaque URLs, where it's not clear from the URL itself what it is pointing to. As to implementation, I suppose you could do something like:

```lua
bookmarks = {
    ["https://some.nested/website"] = "Description",
    "https://github.com/lalitmee/browse.nvim",
}
```

or even using a nested table:

```lua
bookmarks = {
    { ["url"] = "https://some.nested/website", ["description"] = "Some description" },
    { ["url"] = "https://github.com/lalitmee/browse.nvim" },
}
```

**Comment by lalitmee** [comment]
Implemented in 3336967066f57af56e6d2a8c07ac09acce351cd5
```lua
local bookmarks = {
  ["github"] = {
      ["name"] = "search github from neovim", -- description of the group
      ["code_search"] = "https://github.com/search?q=%s&type=code",
      ["repo_search"] = "https://github.com/search?q=%s&type=repositories",
      ["issues_search"] = "https://github.com/search?q=%s&type=issues",
      ["pulls_search"] = "https://github.com/search?q=%s&type=pullrequests",
  },
}
```
